Assets/Scripts/BlackHoleExpansion.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+
+public class BlackHoleExpansion : MonoBehaviour
+{
+    
+    public float expansionRate = 0.5f; // the fraction of current size to increase by per second
+
+    private void Update()
+    {
+        transform.localScale *= expansionRate * Time.deltaTime + 1;
```

Comment:
I'd rename this all to `BlackHoleController` and put this expansion stuff in an `Expand` function. We might want to later have more complex interactions with the blackhole.

Reply:
Why? What about separation of concerns?